applications/matter_bridge/doc/adding_ble_bridged_device_service.rst

```diff
@@ -0,0 +1,258 @@
+
+        #include "my_bt_service.h"
+
+        class MyBtServiceDataProvider : public BLEBridgedDeviceProvider {
+        public:
+            explicit MyBtServiceDataProvider(UpdateAttributeCallback updateCallback, InvokeCommandCallback commandCallback) : BLEBridgedDeviceProvider(updateCallback, commandCallback) {}
+
+        };
+
+   #. Open the :file:`nrf/samples/matter/common/bridge/ble_bridged_device.h` header file again to see which methods of :c:struct:`BLEBridgedDeviceProvider` class are purely virtual (assigned with ``=0``) and have to be overridden by the :c:struct:`MyBtServiceDataProvider` class.
+      Note that :c:struct:`BLEBridgedDeviceProvider` inherits from the :c:struct:`BridgedDeviceDataProvider` class, so the :c:struct:`MyBtServiceDataProvider` class has to implement the purely virtual methods of :c:struct:`BridgedDeviceDataProvider` as well.
+   #. Edit the :c:struct:`MyBtServiceDataProvider` class, in the :file:`my_bt_service_data_provider.h` header file to declare the required methods as follows:
+
+      .. code-block:: C++
+
+        void Init() override;
+        void NotifyUpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
+                    size_t dataSize) override;
+        CHIP_ERROR UpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer) override;
+        bt_uuid *GetServiceUuid() override;
+        int ParseDiscoveredData(bt_gatt_dm *discoveredData) override;
+
+   #. Implement the body of the :c:func:`Init` method so that it can prepare the data provider for further operation.
+      If there are no additional actions to be done before starting the provider, it can be implemented in the :file:`my_bt_service_data_provider.cpp` file as empty.
+
+      .. code-block:: C++
+
+        void MyBtServiceDataProvider::Init()
+        {
+            /* Do nothing in this case */
+        }
+
+   #. Implement the body of the :c:func:`NotifyUpdateState` method that shall be called after every data change related to the Matter devices bridged to the Bluetooth LE device using ``My Bt Service``.
+      It is used to inform the ``Bridge Manager`` and Matter Data Model that an attribute value should be updated.
+
+      To make the method invoke appropriate callback, edit :file:`my_bt_service_data_provider.cpp` file as follows:
+
+      .. code-block:: C++
+
+        void MyBtServiceDataProvider::NotifyUpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
+                            size_t dataSize)
+        {
+            if (mUpdateAttributeCallback) {
+                mUpdateAttributeCallback(*this, clusterId, attributeId, data, dataSize);
+            }
+        }
+
+   #. Implement the body of the :c:func:`UpdateState` method.
+      This will be called by the ``Bridge Manager`` to inform that data in Matter Data Model was changed and request propagating this information to the Bluetooth LE end device.
```

Comment:
```suggestion
      This will be called by the ``Bridge Manager`` to inform that data in Matter Data Model was changed and request propagating this information to the Bluetooth LE end device.

```